### ISSUE TITLE
Podman fills tmpfs libpod events log

### ISSUE THREAD

**Issue by dewthefifth** [containers/podman#17720]
### Issue Description

Podman is filling the /run/user/{uid}/libpod/tmp/events file when running healthchecks. Specifically the following logs are printed to the event log every check (per container).

{"ID":"1b40395248114496cf1e293e9fc08969b86749cf4fad7a1bcb9ee14677c78ed8","Image":"docker.io/redhat/ubi8:latest","Name":"event-log-full-1","Status":"health_status","Time":"2023-03-08T15:01:31.300060671-05:00","Type":"container","health_status":"healthy","Attributes":{"architecture":"x86_64","build-date":"2023-02-15T04:02:00","com.redhat.component":"ubi8-container","com.redhat.license_terms":"https://www.redhat.com/en/about/red-hat-end-user-license-agreements#UBI","description":"The Universal Base Image is designed and engineered to be the base layer for all of your containerized applications, middleware and utilities. This base image is freely redistributable, but Red Hat only supports Red Hat technologies through subscriptions for Red Hat products. This image is maintained by Red Hat and updated regularly.","distribution-scope":"public","io.buildah.version":"1.27.3","io.k8s.description":"The Universal Base Image is designed and engineered to be the base layer for all of your containerized applications, middleware and utilities. This base image is freely redistributable, but Red Hat only supports Red Hat technologies through subscriptions for Red Hat products. This image is maintained by Red Hat and updated regularly.","io.k8s.display-name":"Red Hat Universal Base Image 8","io.openshift.expose-services":"","io.openshift.tags":"base rhel8","maintainer":"Red Hat, Inc.","name":"ubi8","release":"1090","summary":"Provides the latest release of Red Hat Universal Base Image 8.","url":"https://access.redhat.com/containers/#/registry.access.redhat.com/ubi8/images/8.7-1090","vcs-ref":"a995512a05037e3b60bbb1bf9fa6e394063131c3","vcs-type":"git","vendor":"Red Hat, Inc.","version":"8.7"}}
{"ID":"1b40395248114496cf1e293e9fc08969b86749cf4fad7a1bcb9ee14677c78ed8","Image":"docker.io/redhat/ubi8:latest","Name":"event-log-full-1","Status":"exec_died","Time":"2023-03-08T15:01:31.328580501-05:00","Type":"container","Attributes":{"execID":"4607473ac4dbdb9f76a4262c0b543270bb1fa3871042f13352d1bc79cc208847"}}

### Steps to reproduce the issue

Steps to reproduce the issue
```
podman network create event-log-full

podman pod create --name event-log-full --network event-log-full --share net
podman run \
         --name=event-log-full-1 -d \
         --pod=event-log-full \
         --health-cmd="echo hello" \
         docker.io/redhat/ubi8 tail -f /dev/null
```

Then monitor the event log
```
tail /run/user/100141/libpod/tmp/events/events.log -f
```

### Describe the results you received

Eventually the events file grows too large and podman stops being able to start/stop/interact with pods correctly due to the disk being full. For me, this is currently a 3GB file.

### Describe the results you expected

Continued, indefinite execution of a pod

### podman info output

```yaml
host:
  arch: amd64
  buildahVersion: 1.27.3
  cgroupControllers: []
  cgroupManager: cgroupfs
  cgroupVersion: v1
  conmon:
    package: conmon-2.1.4-1.module+el8.7.0+17498+a7f63b89.x86_64
    path: /usr/bin/conmon
    version: 'conmon version 2.1.4, commit: 419a7c7817d7da098aa7648abddea5014f593ac6'
  cpuUtilization:
    idlePercent: 97.11
    systemPercent: 1.18
    userPercent: 1.71
  cpus: 12
  distribution:
    distribution: '"rhel"'
    version: "8.7"
  eventLogger: file
  hostname: GJ286D3
  idMappings:
    gidmap:
    - container_id: 0
      host_id: 510
      size: 1
    - container_id: 1
      host_id: 200000
      size: 65536
    uidmap:
    - container_id: 0
      host_id: 100141
      size: 1
    - container_id: 1
      host_id: 200000
      size: 65536
  kernel: 4.18.0-425.3.1.el8.x86_64
  linkmode: dynamic
  logDriver: k8s-file
  memFree: 1692635136
  memTotal: 66941595648
  networkBackend: cni
  ociRuntime:
    name: runc
    package: runc-1.1.4-1.module+el8.7.0+17498+a7f63b89.x86_64
    path: /usr/bin/runc
    version: |-
      runc version 1.1.4
      spec: 1.0.2-dev
      go: go1.18.4
      libseccomp: 2.5.2
  os: linux
  remoteSocket:
    path: /run/user/100141/podman/podman.sock
  security:
    apparmorEnabled: false
    capabilities: CAP_NET_RAW,CAP_CHOWN,CAP_DAC_OVERRIDE,CAP_FOWNER,CAP_FSETID,CAP_KILL,CAP_NET_BIND_SERVICE,CAP_SETFCAP,CAP_SETGID,CAP_SETPCAP,CAP_SETUID,CAP_SYS_CHROOT
    rootless: true
    seccompEnabled: true
    seccompProfilePath: /usr/share/containers/seccomp.json
    selinuxEnabled: true
  serviceIsRemote: false
  slirp4netns:
    executable: /usr/bin/slirp4netns
    package: slirp4netns-1.2.0-2.module+el8.7.0+17498+a7f63b89.x86_64
    version: |-
      slirp4netns version 1.2.0
      commit: 656041d45cfca7a4176f6b7eed9e4fe6c11e8383
      libslirp: 4.4.0
      SLIRP_CONFIG_VERSION_MAX: 3
      libseccomp: 2.5.2
  swapFree: 33400549376
  swapTotal: 33641459712
  uptime: 144h 59m 10.00s (Approximately 6.00 days)
plugins:
  authorization: null
  log:
  - k8s-file
  - none
  - passthrough
  - journald
  network:
  - bridge
  - macvlan
  - ipvlan
  volume:
  - local
registries:
  search:
  - registry.access.redhat.com
  - registry.redhat.io
  - docker.io
store:
  configFile: /home/$USERNAME/.config/containers/storage.conf
  containerStore:
    number: 2
    paused: 0
    running: 2
    stopped: 0
  graphDriverName: overlay
  graphOptions: {}
  graphRoot: /home/$USERNAME/.local/share/containers/storage
  graphRootAllocated: 972172300288
  graphRootUsed: 231634817024
  graphStatus:
    Backing Filesystem: extfs
    Native Overlay Diff: "true"
    Supports d_type: "true"
    Using metacopy: "false"
  imageCopyTmpDir: /var/tmp
  imageStore:
    number: 147
  runRoot: /run/user/100141/containers
  volumePath: /home/$USERNAME/.local/share/containers/storage/volumes
version:
  APIVersion: 4.2.0
  Built: 1670845316
  BuiltTime: Mon Dec 12 06:41:56 2022
  GitCommit: ""
  GoVersion: go1.18.4
  Os: linux
  OsArch: linux/amd64
  Version: 4.2.0
```

### Podman in a container

No

### Privileged Or Rootless

Rootless

### Upstream Latest Release

No

**Comment by Luap99** [comment]
Podman should rotate the event log file since v4.1, the size can be set in containers.conf.
Check `events_logfile_max_size`, it should default to 1 MB. Can you check your containers.conf files to see if this is set to 0 (unlimited) for some reason?

**Comment by dewthefifth** [comment]
I apologize. I confirmed the content was being written to the events file using my dev machine, which is also where I acquired the actual podman info from. However, my production machine (which is the machine that actually filled up) looks to be using 4.0.2 instead of 4.2.0 and I didn't notice when I was comparing the two. I've updated the production machine's podman version and am closing the ticket.

Thank you
